src/Assets/Javascript/fetch.js

```diff
@@ -0,0 +1,9 @@
+export default function fetchPics(api, array1, array2) {
```

Comment:
* You still have the `function` keyword used here instead of the `arrow` function ES6 syntax
 * I suggest you use the ES6 `arrow` function in place of the `function` keyword this would make your code look more modern and similar to code bases out there in the wild so it's better for you to practice using ES6 syntax as early as possible 👍